Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheetProtocol.swift

```diff
@@ -432,6 +430,15 @@ extension PhotonActionSheetProtocol {
         return openDownloads
     }
 
+    private func collectionItem(vcDelegate: PageOptionsVC, openHomePage: PhotonActionSheetItem) -> PhotonActionSheetItem {
```

Comment:
I would rather not have this function at all as it is being used only once and it's interface is confusing. Could you please move the implementsation directly to the where it is being used?

Reply:
Done.